### PR TITLE
Robot description for Jackal

### PR DESCRIPTION
**Pull request by mikepurvis**
Description as written is missing mass and inertial numbers. These will need to be filled in correctly when the simulator is developed.

There's a test rviz launcher committed [here](https://github.com/jackal/jackal_desktop/blob/indigo-devel/jackal_viz/launch/view_model.launch).

@rgariepy for review.

### CODE REVIEW

**Comment by mikepurvis** [comment]
Now includes accessory mount points

**Comment by mikepurvis** [comment]
Now includes the navsat and imu links. @y22ma, @p6chen, I'd appreciate if either of you could give a quick thumbup on this as well, as you're the end consumers of it.

File: jackal_description/launch/description.launch [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<launch>
+  <param name="robot_description" command="$(find xacro)/xacro '$(find jackal_description)/urdf/jackal.urdf.xacro'" />
+  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
```
Comment (y22ma) [inline jackal_description/launch/description.launch]: state_publisher is actually deprecated, please use robot_state_publisher in the type tag.

Comment (mikepurvis) [inline jackal_description/launch/description.launch]: Good call, fixed.

File: jackal_description/urdf/jackal.urdf.xacro [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+  <!-- TODO: Make this internal_imu_link or something, and use a mixed-in xacro
+       to supply the joint between it and imu_link. This is important so that imu_link
+       always points to the "active" IMU. When an upgrade IMU is connected, the
+       internal_imu_link should remain, but imu_link should point to the upgrade one. -->
+  <link name="imu_link"></link>
+  <joint name="imu_joint" type="fixed">
+    <parent link="base_link" />
+    <child link="imu_link" />
+    <origin xyz="-0.1974 0.0314 0.163" />
```
Comment (y22ma) [inline jackal_description/urdf/jackal.urdf.xacro:100]: Should there be a RPY tag here to specify its mounting orientation? What is the standard IMU mounting pose on Jackal?

Comment (mikepurvis) [inline jackal_description/urdf/jackal.urdf.xacro:100]: The gyro/accel/compass data will be transformed in firmware so that the axes match base_link.

Comment (y22ma) [inline jackal_description/urdf/jackal.urdf.xacro:100]: Ok.

**Comment by y22ma** [comment]
LGTM

**Comment by mikepurvis** [comment]
@rgariepy ok?

**Comment by rgariepy** [comment]
Didn't get a chance to review. OK with Yan's OK.

**Pull request merged**